geniza/corpus/templates/corpus/add_transcription_source.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'base.html' %}
+
+{% load admin_urls static render_bundle_csp %}
+
+{% block meta_title %}{{ page_title }}{% endblock meta_title %}
+
+{% block extrascript %}
+    {# jQuery required for autocomplete widget #}
+    <script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.min.js' %}"></script>
+    {% render_bundle_csp "annotation" "js" %}
+{% endblock %}
+
+{% block main %}
+    <h1>{{ page_title }}</h1>
+    <h2>Select source</h2>
+
+    {{ formset.media }}
+    <form method="post" id="add-transcription-source">
+        {% csrf_token %}
+        <fieldset>
+            {{ formset }}
+        </fieldset>
+        <button class="primary" type="submit">Create transcription</button>
+    </form>
+
```

Comment:
I don't think it should be MVP, but I'm curious: how difficult would it be to search for an unpublished source where the currently logged in user is the sole author? Could we provide a convenience link for that source in addition to this form?

Reply:
I think that's a great idea. I wonder if we should actually make it a one-to-one relation on their user account, to remove the guesswork; if there isn't already one, this could be a place to prompt the user to create one.

Either way, I'll add a TODO here.

Reply:
You know, I think ages ago we discussed having an optional 1-1 between `Creator` and `User` since some of them are the same people. We didn't have a reason to do it before, but implementing this feature would be a logical time to add it.

Reply:
Split out into separate issue, thanks!